Resources/Prototypes/_Sunrise/tts_voices_sunrise.yml

```
@@ -893,15 +893,21 @@
+  sex: 
+  - Male
+  #Lust-End
   speaker: hearthstone_bob1
   provider: sunrise
   sponsorOnly: false
 
 - type: ttsVoice
   id: HeartstoneRexar
   name: tts-voice-name-heartstone-rexar
-  sex: Male
+  #Lust-Start
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)